conans/test/functional/configuration/profile_test.py

```diff
@@ -518,7 +518,7 @@ def setUp(self):
     def test_create(self):
 
         # The latest declared profile has priority
-        self.client.run("create . lib/1.0@user/channel --profile profile1 -p profile2")
+        self.client.run("create . lib/1.0@user/channel --profile profile1 -pr profile2")
```

Comment:
These are unrelated, better do them in a previous PR

Reply:
These are related to this PR, with this new `*_build/*_host` (and their short forms), the command line is not able to disambiguate the _wrong_ `-p` argument and fails.

I think that `-p` should be an error, but it works, only this PR is _breaking_ that existing behavior.